packages/block-editor/src/components/url-input/style.scss

```diff
@@ -30,6 +25,15 @@ $input-size: 300px;
 	}
 }
 
+// Only apply min-width to URL inputs within popovers or block list blocks
+.block-editor-block-list__block .block-editor-url-input,
```

Comment:
I'm not sure of the context as to why we need `.block-editor-block-list__block` here 🤔